libsignal-service/src/sender.rs

```diff
@@ -663,7 +663,7 @@ where
 
         // When sending to ourselves, don't include the local device
         if myself {
-            devices.retain(|id| *id != self.device_id.into());
+            devices.retain(|id| DeviceId::from(*id) != self.device_id);
```

Comment:
For some reason, this comparison started complaining when I implemented the `SignalWebSocket::retrieve_profile_by_id` method. Commenting it out, it worked, but including it triggered some shenanigans around the `.into()` call needing disambiguation.